<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaptureWidget</class>
 <widget class="QWidget" name="CaptureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="enabledCheckBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Capture from file or camera</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="sourceType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>611</width>
     <height>221</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Camera</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera number</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="cameraNumberSpinBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>37</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Image</string>
    </attribute>
    <widget class="QPushButton" name="file1Button">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>40</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Take screenshot from the current filter's output window. Save as CV_8U single-channel PNG.</string>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Image file</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file1Name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>501</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="file2Button">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Take screenshot from the current filter's output window. Save as CV_8U single-channel PNG.</string>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>132</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Second image file (optional)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file2Name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>501</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Video file</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="videoFileName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>501</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="videoFileButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>40</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Take screenshot from the current filter's output window. Save as CV_8U single-channel PNG.</string>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QSlider" name="videoProgress">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>501</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelFrames">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current frame:  0 of 100</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Filled</string>
    </attribute>
    <widget class="QSpinBox" name="frameWidthSpinBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1920</number>
     </property>
     <property name="value">
      <number>640</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>96</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Size of empty frame</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="frameHeightSpinBox">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1080</number>
     </property>
     <property name="value">
      <number>480</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>237</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>(Empty frame has type CV_8UC3 and filled white)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QSpinBox" name="refreshPeriodMs">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>33</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>93</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame interval (ms)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
